Source/ACE/Entity/ModelData.cs

```diff
@@ -11,13 +11,13 @@ namespace ACE.Entity
     /// </summary>
```

Comment:
/// TODO: convert to using AceObject backed data and get rid of the redundant copies of all the datum.